main.go

```diff
@@ -72,6 +72,10 @@ func main() {
 		os.Exit(1)
 	}
 
+	if v, found := os.LookupEnv("OCI_REGISTRY_LOCALHOST"); found {
```

Comment:
What is the purpose of this? 🤔  Local testing? Or was this just left in?

Reply:
It's useful for local testing.